docs/windows_dev.md

```diff
@@ -0,0 +1,101 @@
+# Integration Tests: Windows Local Setup
+**Prerequisites**: 
+- You've setup Go and the buildkit repo and you can build from source.
+- Optional: You have installed the [Delve debugger](https://github.com/go-delve/delve)
+
+### Main Steps
+- Install `registry.exe` by running the following command:
+
+  ```go install -v github.com/distribution/distribution/v3/cmd/registry@latest```
+- Install buildkit with the following command: 
+     ```
+    cd to repo directory
+    go install -v .\cmd\...
+    ```
```

Comment:
I don't think this is necessary, you can just run:

```
docker buildx bake binaries-for-test --set *.platform=windows/amd64
```

To output dev binaries:

```
$ tree -nh ./bin/build
[4.0K]  ./bin/build
├── [ 29M]  buildctl.exe
├── [ 56M]  buildkitd.exe
├── [ 17M]  containerd-shim-runhcs-v1.exe
├── [ 37M]  containerd.exe
├── [2.2M]  gocovmerge.exe
├── [7.5M]  gotestsum.exe
├── [ 573]  gotestsumandcover
└── [ 23M]  registry.exe

0 directories, 8 files
```

Will open a PR to also include delve